js/theme/components/Text.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import { type Theme } from '../types'
+import variables from '../variables'
+
+export default (): Theme => {
+  const theme = {
+    '.link': {
+      color: variables.textLinkColor,
+      fontWeight: '600'
```

Comment:
shouldn't this come from `variables`? it's a pretty core settings...

Reply:
Yes better.